azure-pipelines/build-python-packages.yml

```diff
@@ -13,7 +13,7 @@ stages:
   - template: /azure-pipelines/templates/build-job.yml
 
 - stage: Test_Python_MacOS
-  condition: succeeded()
+  condition: and(succeeded(),or(contains(variables['VERSION'], 'alpha'),contains(variables['VERSION'], 'beta')))
```

Comment:
I don't think that we should skip tests for alpha and beta versions fully because we won't be able to prove that built Python works.
Looks like 3.10.0a6 has issues with building single module (tkinter), can we just implement exception for it?